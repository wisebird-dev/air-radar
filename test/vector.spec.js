import React from 'react';
import {shallow} from 'enzyme';

import Vector from '../src/components/vector';

test('it works', () => {
	const wrapper = shallow(<Vector/>);
	expect(wrapper).toMatchSnapshot();
});
