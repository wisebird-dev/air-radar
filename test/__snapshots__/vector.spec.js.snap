// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    width: 18px;
    height: 18px;
    margin-right: 10px;
    vertical-align: text-top;
",
        ],
      },
      "displayName": "styled.img",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "img",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
